code/Base/GamemodeSystem/Entities/BaseGamemode.cs

```diff
@@ -94,6 +95,12 @@ public void RemoveClient( Client cl, LeaveReason reason = LeaveReason.Leave )
 		OnClientRemoved( cl, reason );
 	}
 
+	public void AddBot( SportsBot bot )
```

Comment:
I think this is kinda backwards. AddClient should check if it's a bot, and then run OnBotAdded as well.

Reply:
i actually disagree with this. Bot's client shouldn't be handled any different from the normal player clients.
ontop of it making this code a lot more confusing since you wouldn't be able to get the SportsBot class as a result.

Tbh not even sure your idea would work. since Sportsbot isn't linked to the Client of the bot